handler.go

```diff
@@ -56,8 +56,9 @@ type Handler struct {
 	StatusHandler StatusHandler
 
 	// Local hostname & cluster configuration.
-	Host    string
+	Host    *URI
```

Comment:
Can we rename the Host variable to `URI`?

Reply:
ok